code/game/objects/items/crayons.dm

```diff
@@ -754,12 +754,15 @@
 			var/list/skins = list()
 			var/static/list/style_list_icons = list("standard" = 'icons/mob/augmentation/augments.dmi', "engineer" = 'icons/mob/augmentation/augments_engineer.dmi', "security" = 'icons/mob/augmentation/augments_security.dmi', "mining" = 'icons/mob/augmentation/augments_mining.dmi')
 			for(var/skin_option in style_list_icons)
-				var/image/part_image = image(icon = style_list_icons[skin_option], icon_state = limb.icon_state)
+				var/image/part_image = image(icon = style_list_icons[skin_option], icon_state = "[limb.limb_id]_[limb.body_zone]")
+				if(limb.aux_zone)
+					part_image.overlays += image(icon = style_list_icons[skin_option], icon_state = "[limb.limb_id]_[limb.aux_zone]")
 				skins += list("[skin_option]" = part_image)
 			var/choice = show_radial_menu(user, src, skins, require_near = TRUE)
 			if(choice && (use_charges(user, 5, requires_full = FALSE) == 5))
 				playsound(user.loc, 'sound/effects/spray.ogg', 5, TRUE, 5)
-				limb.icon = style_list_icons[choice]
+				limb.icon_static = style_list_icons[choice]
```

Comment:
this entire thing should be moved onto `/obj/item/bodypart/x/robotic`, and adding raw overlays to bodyparts is bad.

Reply:
This sounds a bit nightmarish to figure out with how the robotic bodyparts types are spead out, so I will try to add a solution to the base class, which could be useful in the future for like, species that turn their skin into shiny armour at will or something.
I am unsure how I should go about swapping from icon_static (though now i see, it is static for a reason.). Would an associate list of alternate skins work, along with a var that stores the key of the selected alternate skin, and use that during updating the limb icon, if not null?

Reply:
i am at work but i can tell you the ideal solution on discord in about 5 hours, Kapu#1178 im in tg's discord

Reply:
Thank you. Though I will be asleep at that time, I will look for you later.

Reply:
Looks like time zones has struck again, you are probably asleep by this time... I am ProfessorLizzard#0245 on discord, in case you notice me being online before i notice you. :)

Reply:
I decided to go and do it myself since there's some code cleanup I've wanted to do anyway https://github.com/tgstation/tgstation/pull/69841

Reply:
Thank you, I will convert this PR to only be about the bleeding cyborg eyes.